src/main/java/org/opentripplanner/middleware/triptracker/TravelerLocator.java

```diff
@@ -57,14 +58,14 @@ public static String getInstruction(
     ) {
         if (hasRequiredWalkLeg(travelerPosition)) {
             if (hasRequiredTripStatus(tripStatus)) {
-                TripInstruction tripInstruction = alignTravelerToTrip(travelerPosition, isStartOfTrip, tripStatus);
+                TripInstruction tripInstruction = alignTravelerToTrip(travelerPosition, isStartOfTrip);
                 if (tripInstruction != null) {
                     return tripInstruction.build();
                 }
             }
 
             if (tripStatus.equals(TripStatus.DEVIATED)) {
-                TripInstruction tripInstruction = getBackOnTrack(travelerPosition, isStartOfTrip, tripStatus);
+                TripInstruction tripInstruction = getBackOnTrack(travelerPosition, isStartOfTrip);
```

Comment:
Bonus points question: I didn't see an equivalent to "Head to <departure bus stop>" on an itinerary where the first leg is bus and tracking is started at a location away from the bus stop.

Reply:
@binh-dam-ibigroup Update here: https://github.com/ibi-group/otp-middleware/pull/264/commits/eeea56917235f4b7c431e1b58937d23c242c334f

Reply:
@binh-dam-ibigroup I will hold off merging until you have "approved" this addition.